syft/frameworks/torch/he/fv/plaintext.py

```diff
@@ -0,0 +1,26 @@
+import torch as th
+from syft.frameworks.torch.he.fv.util.operations import get_significant_count
+
+
+class PlainText:
+    """A wrapper class for representing plaintext elements.
+
+    Attributes:
+        data: A list of values of plaintext polynomial.
+    """
+
+    def __init__(self, data):
+        self._data = data
+        self._coeff_count = len(data)
+
+    def significant_coeff_count(self):
+        """Returns the significant coefficient count of the polynomial (removes leading zeroes)"""
+        return get_significant_count(self._data)
+
+    @property
```

Comment:
This pattern shows up way more in this PR than I've seen anywhere else in the code base. What's the intended purpose?

Reply:
it is just a wrapper class. I am using for making the data(list) more meaningful.

Reply:
I like the wrapper class. 👍